packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Internal/Optional.cs

```diff
@@ -35,14 +35,14 @@ public static bool IsDefined(object value)
             return value != null;
         }
 
-        public static bool IsDefined(JsonElement value)
+        public static bool IsDefined(string value)
```

Comment:
is this change expected?

Reply:
Yeah, since we are overriding the Optional provider's `BuildMethod` by calling base and adding the `IsJsonElementDefined` method to it I expected it would change the order of it. It is still generating the same methods.

Reply:
oh, i see its just an order change, thought it was a replacement